### PR TITLE
A11y: Sort dropdown ARIA label

### PR DESCRIPTION
**Pull request by lhenze**
Adding possibility for accessible name to the search results sort dropdown

### CODE REVIEW

File: src/lib/components/Sort/Sort.js [hunk 1]
```diff
@@ -83,6 +86,7 @@ Sort.propTypes = {
   label: PropTypes.func,
   overridableId: PropTypes.string,
   sortOrderDisabled: PropTypes.bool,
+  ariaLabel: PropTypes.string
```
Comment (ntarocco) [inline src/lib/components/Sort/Sort.js:89]: can you please add the `default` value just below, given that it is not a required `prop`?

**Review by ntarocco — COMMENTED** [review]
Thanks for the contribution!
Are you planning to add the `ARIA` tags to more components?

File: src/lib/components/Sort/Sort.js (hunk 1 above)
Comment (lhenze) [reply src/lib/components/Sort/Sort.js:89]: Empty string is not great for A11y.  I put in the default value of "Sort" even though it's not translated.  Will also make a PR for invenio-search-ui that includes translation option.

**Review by NRodriguezcuellar — APPROVED** [review]

**Review by kpsherva — COMMENTED** [review]
It looks good to me!
Could you squash the commits to one and edit the commit message in order to follow the invenio contribution guidelines?

**Comment by lhenze** [comment]
Commits have been squashed and message amended.

**Pull request merged**